package.json

```diff
@@ -50,12 +51,12 @@
     }
   },
   "devDependencies": {
-    "@commitlint/cli": "^5.2.8",
+    "@commitlint/cli": "^6.0.2",
     "commitizen": "^2.9.6",
     "commitlint-config-seek": "^1.0.0",
     "cz-conventional-changelog": "^2.1.0",
     "husky": "^0.14.3",
-    "renovate": "10.46.0",
+    "renovate": "10.49.3",
```

Comment:
Not strictly related to this pr but should this be added as a peerDependency as-well?

Reply:
This is only here for testing purposes, to ensure our config is actually valid. Consumers don't need Renovate to be installed.